src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -49,9 +49,11 @@
 import org.bukkit.scheduler.BukkitTask;
 
 import java.io.File;
+import java.util.regex.Pattern;
 
 public class FunnyGuilds extends JavaPlugin {
 
+    public static final Pattern USERNAME_PATTERN = Pattern.compile("^[A-Za-z0-9_]{3,16}$");
```

Comment:
Username pattern is related to the User domain classes, so it shouldn't be created outside of its package